File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/LowLevelFields_View.cpp

#include "LowLevelFields_View.h"
#include "LowLevelFields_View.h"
#include <vector>
#include <memory>
#include "LowLevelFields_ViewCustomTableRowFieldRow.h"
#include "LowLevelFields_ViewCustomListRowFieldRow.h"
#include "LowLevelFields_ViewCustomTreeRowFieldRow.h"
#include <string>

namespace widgetassertions
{
  bool LowLevelFields_View::getMyBool( ) 
  {
    return this->MyBool;
  }
  int LowLevelFields_View::getMyInt( ) 
  {
    return this->MyInt;
  }
  std::string LowLevelFields_View::getMyString( ) 
  {
    return this->MyString;
  }
  std::vector<std::string> LowLevelFields_View::getMyStrings( ) 
  {
    return this->MyStrings;
  }
  std::vector<std::shared_ptr<LowLevelFields_ViewCustomTableRowFieldRow>> LowLevelFields_View::getCustomTableRowFieldTableRows( ) 
  {
    return this->CustomTableRowFieldTableRows;
  }
  std::vector<std::shared_ptr<LowLevelFields_ViewCustomListRowFieldRow>> LowLevelFields_View::getCustomListRowFieldListRows( ) 
  {
    return this->CustomListRowFieldListRows;
  }
  std::vector<std::shared_ptr<LowLevelFields_ViewCustomTreeRowFieldRow>> LowLevelFields_View::getCustomTreeRowFieldTreeRows( ) 
  {
    return this->CustomTreeRowFieldTreeRows;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/LowLevelFields_View.cs
/// <filename>
///     LowLevelFields_View.cs
/// </filename>
using System.Collections.Generic;
using widgetassertions;

namespace widgetassertions
{
    public class LowLevelFields_View
  {
        public bool MyBool;
     
        public     virtual bool getMyBool()
    {
      return this.MyBool;
    }
     
        public int MyInt;
     
        public     virtual int getMyInt()
    {
      return this.MyInt;
    }
     
        public string MyString;
     
        public     virtual string getMyString()
    {
      return this.MyString;
    }
     
        public System.Collections.Generic.List<string> MyStrings = new System.Collections.Generic.List<string>();
     
        public     virtual System.Collections.Generic.List<string> getMyStrings()
    {
      return this.MyStrings;
    }
     
        private System.Collections.Generic.List<widgetassertions.LowLevelFields_ViewCustomTableRowFieldRow> CustomTableRowFieldTableRows = new System.Collections.Generic.List<widgetassertions.LowLevelFields_ViewCustomTableRowFieldRow>();
     
        public     virtual System.Collections.Generic.List<widgetassertions.LowLevelFields_ViewCustomTableRowFieldRow> getCustomTableRowFieldTableRows()
    {
      return this.CustomTableRowFieldTableRows;
    }
     
        private System.Collections.Generic.List<widgetassertions.LowLevelFields_ViewCustomListRowFieldRow> CustomListRowFieldListRows = new System.Collections.Generic.List<widgetassertions.LowLevelFields_ViewCustomListRowFieldRow>();
     
        public     virtual System.Collections.Generic.List<widgetassertions.LowLevelFields_ViewCustomListRowFieldRow> getCustomListRowFieldListRows()
    {
      return this.CustomListRowFieldListRows;
    }
     
        private System.Collections.Generic.List<widgetassertions.LowLevelFields_ViewCustomTreeRowFieldRow> CustomTreeRowFieldTreeRows = new System.Collections.Generic.List<widgetassertions.LowLevelFields_ViewCustomTreeRowFieldRow>();
     
        public     virtual System.Collections.Generic.List<widgetassertions.LowLevelFields_ViewCustomTreeRowFieldRow> getCustomTreeRowFieldTreeRows()
    {
      return this.CustomTreeRowFieldTreeRows;
    }
  }
}

